src/Terminal/commands/common/editor.ts

```diff
@@ -47,10 +49,17 @@ export function commonEditor(
     if (!hasScriptExtension(path) && !hasTextExtension(path)) {
       return Terminal.error(`${command}: Only scripts or text files can be edited. Invalid file type: ${arg}`);
     }
+    if (path.endsWith(".script")) hasNs1 = true;
     const file = server.getContentFile(path);
     const content = file ? file.content : isNs2(path) ? newNs2Template : "";
     files.set(path, content);
     if (content === newNs2Template) CursorPositions.saveCursor(path, { row: 3, column: 5 });
   }
+  if (hasNs1) {
+    Terminal.warn(
+      "NOTICE: NS1 (.script) scripts are deprecated and will be removed in a future update." +
+      " Migrate to NS2 (.js) scripts instead."
```

Comment:
Please add a comment to seek the migration guide in the documentation, ie.
" Migrate to NS2 (.js) scripts following the migration guide in the documentation."